pkg/ipamd/ipamd.go

```diff
@@ -134,6 +134,8 @@ const (
 
 	// trunkInterfaceLabel is used by CNI to check if correct label was added by the VPC resource controller
 	trunkInterfaceLabel = "vpc.amazonaws.com/has-trunk-attached"
+	// VPC resource controller label nodes as not-supported if the EC2 instance is not supported yet
```

Comment:
nit: `if the EC2 instance is not supported yet` --> `if the EC2 instance doesn't support Branch ENI feature`

Reply:
Makes sense. Updated the comment.